src/views/CityExplorer/Weather.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+
+export class Weather extends Component {
+  
+  render() {
+    console.log('props weather: ', this.props.weather['city_name']);
+    return (
+      <div>
+      <h3>Weather in {this.props.location.search_query}</h3>
+        {this.props.weather &&
+        <div>
```

Comment:
this div is next to the h3 and should not be indented

Reply:
> this div is next to the h3 and should not be indented

It actually should be indented, because it's inside the `{this.props.weather ...}` expression. Idiomatic React formatting would look like this:

```js
      <div>
        <h3>Weather in {this.props.location.search_query}</h3>
        {this.props.weather &&
          <div>
```

Thanks for another great example of inconsistent indentation making code harder to read. 😀